mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) [2017] Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
The package directory structure does not match the package name (there is nazgul in the folder name...)

Reply:
Thanks for fixing.